python-checks/src/main/java/org/sonar/python/checks/ChangeMethodContractCheck.java

```diff
@@ -43,6 +43,7 @@
 public class ChangeMethodContractCheck extends PythonSubscriptionCheck {
 
   private static final Set<String> IGNORING_DECORATORS = Set.of(
+    "abc.abstractmethod",
     "abstractmethod",
```

Comment:
Can you add (either in `ProjectLevelTypeTableTest` or `ProjectLevelSymbolTableTest`, or any relevant place) a test that highlights the discrepancies between decorator properly resolved from Typeshed (where I assume we only get a local name) and the case where a proper FQN is resolved?
We should also have a ticket to track this.